### PR TITLE
integration: add test cases for setting update via configmap

### PR DESCRIPTION
**Pull request by derekbit**
longhorn/longhorn#2570

Signed-off-by: Derek Su <derek.su@suse.com>

### CODE REVIEW

**Review by kaxing — DISMISSED** [review]
Looking good!

**Review by cchien816 — APPROVED** [review]
LGTM

**Comment by cchien816** [comment]
e2e result on v1.3.0-rc2 : [Pass](https://ci.longhorn.io/job/private/job/longhorn-tests-regression/1438/testReport/tests/test_settings/)

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, just one comment needed to resolve.

File: manager/integration/tests/test_settings.py [hunk 1]
```diff
@@ -782,3 +786,190 @@ def test_setting_concurrent_rebuild_limit():  # NOQA
...
+
+    # Step 2
+    init_longhorn_default_setting_configmap(core_api, client)
+
+    # Step 3
+    assert wait_for_setting_updated(client,
+                                    SETTING_DEFAULT_REPLICA_COUNT,
+                                    old_setting.value)
+    # Step 4
+    num_threads = 50
```
[the remark this answers is not in the document]
Comment (derekbit) [reply manager/integration/tests/test_settings.py]: I remove the concurrent test from e2e, because it should be at scaling test suite.

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
@derekbit Please help backport to 1.3.x and 1.2.x. Thanks.

**Pull request merged**